docs/RETRY_POLICY.md

```diff
@@ -0,0 +1,29 @@
+# Retry Policy
+
+The Retry Policy configures the connection retry behavior of ecChronos with Cassandra. The Timeout specifies the period of time ecChronos will wait for Cassandra to start per attempt. The retry policy sets the maximum number of connection attempts (maxAttempts), which by default is 5, specifying the time unit as seconds by deafult. Additionally, it determines the delay between each connection attempt (delay), multiplied by the current attempt count.
+
+If the calculated delay exceeds the allowed maximum (maxDelay), the maximum will be used. The maxDelay limits the maximum time a connection attempt can wait to execute. If set to 0, it disables maxDelay, and the delay interval will be calculated based on the number of attempts and the default delay. These settings aim to optimize the efficiency of connection attempts, ensuring the system waits an appropriate amount of time and makes a reasonable number of attempts before giving up.
+
```

Comment:
Perhaps you could write an example here of a "timeline" of the connection attempts to visualize how the settings actually affects the delays/timeout? You know, if you pretend that all connection attempts fails. I think that would be good, because it easily gets just a lot of number juggling otherwise. ;-)